lib/dialyzer/src/dialyzer_codeserver.erl

```diff
@@ -98,9 +98,9 @@
 
 %% We KNOW that `error` is not a valid value in the table.
```

Comment:
This comment needs to be changed I think. But why change `error` to `'$missing'` at all?